eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -108,6 +108,8 @@ jobs:
 
       - script: |
           set -o pipefail
+          xcrun simctl shutdown all
```

Comment:
Did we determine if these are necessary to make the tests work? In theory they shouldn't since we should always get a fresh VM / simulator every run, so if we don't need them we should back this change out.

Reply:
I think you are right so I will remove those now